tests/2-config.spec.ts

```diff
@@ -0,0 +1,5 @@
+/*
```

Comment:
Maybe we can add these files until we have actual tests there.

Reply:
The idea to adding these is so if other people come in and help, they don't have to take the decision on how to approach testing, which could lead to different people having different approaches in mind and then having to do extra work with the consolidation of tests and maybe conflicts.

This way each person could potentially just work on a single file without stepping on anyone elses' toes and the standard architecture of the tests is already there.